__tests__/components/Footer/Footer.test.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { render, cleanup, fireEvent } from '@testing-library/react-native';
 import Footer from '../../../src/components/Footer/Footer';
-
+import Store from '../../../src/context/Store';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Store` import has been added but it's not clear from the provided context whether the `Store` is providing necessary context to the `Footer` component. If the `Footer` component or its children use context values, this change is valid. However, if they don't, this import and usage in the wrapper prop are unnecessary.
```diff
- import Store from '../../../src/context/Store';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->